src/oop/special_class_method/hard/q8/QuestionText.md

```diff
@@ -0,0 +1,36 @@
+# oop-special_class_method-hard-q8
+
+## Question Text
+
+You are writing a Java program that details various round objects. Write a Java class called **Circle** that has a
+`double` variable called **radius** and another `String` variable called **colour**. Write a constructor
+for this class that accepts a `double` value and a `String` value, then sets the class variables to those
+values. Also, write a **toString()** that summarizes the attributes of the object. Also, write accessor and mutator
+methods for the necessary variables.
+
+### Circle Class:
+
+Your `Circle` class must have the following items:
+
+- Attributes :
+    - A `double` attribute named **radius** which is not accessible from other classes
+    - A `String` attribute named **colour** which is not accessible from other classes
+
+- Constructors:
+    - A constructor which is accessible from other classes and accepts a`double` and a `String` to initialize the
+      **radius** and **colour** attributes.
+
+- Methods:
+    - a `setRadius` method which accepts a `double` value, updates the *radius** attribute with it, and is accessible
+      from other classes
+    - a `getRadius` method which is accessible from other classes and returns the value of the **radius** attribute
+    - a `setColour` method which accepts an `int` value, updates the **colour** attribute with it, and is accessible
+      from other classes
+    - a `getColour` method which is accessible from other classes and returns the value of the **colour** attribute
+    - a `toString` method which returns a String of the information about the `Circle` class of the form
```

Comment:
change this description to be more like what we discussed in meeting but the rest of the question's toString stuff can stay the same